src/components/framework/title.js

```diff
@@ -39,7 +39,7 @@ class Title extends React.Component {
     const title = "nextstrain";
     return title.split("").map((letter, i) => (
       <span
-        key={i} // eslint-disable-line react/no-array-index-key
```

Comment:
I'm surprised this rule isn't in [the recommended set](https://github.com/jsx-eslint/eslint-plugin-react/blob/master/configs/recommended.js) of react rules. _Missing_ keys have caused a bunch of bugs in auspice before (recently a bug in the legend), and I thought key collisions were just as bad?

Reply:
Pushed 09b5f3f4b1020deb104c2fe8257483187ab3ac3d to enable it.